src/Illuminate/Foundation/Validation/ValidatesRequests.php

```diff
@@ -11,6 +11,13 @@
 trait ValidatesRequests
 {
     /**
+     * Conditional validation rules based on a Closure.
+     *
+     * @var array
+     */
+    protected $sometimes = [];
```

Comment:
I could rename the variable to something more unique, but it's not a problem you can ever fully get rid of. I mean, this trait also sets `$validatesRequestErrorBag`.